src/ocamlorg_web/lib/templates/pages/package_template.eml

```diff
@@ -161,14 +161,19 @@ let render ~readme ~license:_ package =
             <dt class="text-sm font-medium text-gray-500"><%s "Reverse dependencies" %></dt>
             <dd class="mt-2 text-sm text-gray-900 max-h-80 overflow-y-auto">
               <ul class="divide-y divide-gray-100">
-                <% rev_deps |> List.iter begin fun (name, cstr) -> %>
+                <% rev_deps |> List.iter begin fun (name, cstr, version) -> %>
                 <li class="py-2">
                   <a href="/p/<%s name %>" class="mt-1 text-sm text-orange-600 hover:text-orange-500"><%s name %></a>
                   <% match cstr with None -> () | Some cstr -> %>
                   <span class="inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-white text-gray-800">
                     <pre><%s cstr %></pre>
                   </span>
                   <% ; %>
+                  <% match version with None -> () | Some version -> %>
```

Comment:
How can I write 'matching version'?

Reply:
You shouldn't need to -- `version` at this point is a `string`. The match for `cstr` is because it is a `string option`.